package.json

```diff
@@ -85,6 +85,13 @@
     ],
```

Comment:
Above (can't add the comment there) there's an activation event for `onView:azFuncTree` that can be removed.